Softeq.XToolkit.Bindings.iOS/Gestures/GestureRecognizerBehaviour.cs

```diff
@@ -25,13 +26,10 @@ protected void AddGestureRecognizer(UIView target, UIGestureRecognizer tap)
 
             target.AddGestureRecognizer(tap);
         }
-    }
 
-    public abstract class GestureRecognizerBehavior<T>
-        : GestureRecognizerBehavior
-    {
-        protected virtual void HandleGesture(T gesture)
+        protected void HandleGesture(T gesture)
```

Comment:
Maybe we could remove HandleGesture method at all. Or even better - we could move the body of FireCommand method here and remove FireCommand.

Reply:
fixed